testfiles/HVACTemplate-5ZonePurchAir.idf

```diff
@@ -1930,6 +1930,16 @@
 
   Output:Variable,*,Zone Ideal Loads Heat Recovery Active Time,hourly;
 
+  Output:Variable,*,Zone Ideal Loads Supply Air Temperature,Detailed;
+
+  Output:Variable,*,Zone Ideal Loads Supply Air Humidity Ratio,Detailed;
+
+  Output:Variable,*,Zone Ideal Loads Mixed Air Temperature,Detailed;
+
+  Output:Variable,*,Zone Ideal Loads Mixed Air Humidity Ratio,Detailed;
+
+  Output:Variable,*,System Node Mass Flow Rate,Detailed;
+
```

Comment:
Now the results appear in the test file output. Should I change the reporting frequency to hourly? Also, the effects are better visible for an annual simulation.

Reply:
Yes, I would make the reporting frequency consistent - looks like everything else is hourly.